### DISCUSSION TITLE
Limiting the Bot

### DISCUSSION THREAD

**Discussion by flatcap** [neomutt/neo-bot#7] [Ideas]
Currently, `neo-bot` only has one limit:

- Issue/PR must be less than 1 year old (unless spoken to directly)
  This is to prevent false positives, like: `my #1 movie`

We need to consider a few more:

- De-duping.  Occasionally an issue will get mentioned several times in a sentence.
  This causes the bot to repeat itself.

@brunnre8 suggested an improved version of this:

- Cool-off period
  Once an issue has been explained, `neo-bot` won't mention it again (for 5mins, or an hour)

This may or may not be a problem:

- Infinite loops
  Make sure that `neo-bot` never replies to itself
  If the reply to `#1234` has `#1234` in it, will that cause trouble?

**Comment by brunnre8** [comment]
Infinite loops aren't a problem, as the bot doesn't see itself.
